specs/latest/2.0/index.html

```diff
@@ -1274,6 +1274,9 @@ <h2 class="no-toc">Table of contents</h2>
         framebuffer. If the value of SAMPLE_BUFFERS for the read framebuffer is one and the value of
         SAMPLE_BUFFERS for the draw framebuffer is zero, the samples corresponding to each pixel location in
         the source are converted to a single sample before being written to the destination.
+
+        For any source pixel lying outside the read framebuffer, the corresponding destination pixel remains
+        untouched.
```

Comment:
@zhenyao the spec includes special language to say that GL_LINEAR filtering might cause reads outside of the src region in that case.
What we want is for a dest pixel to be written iff it is in the intersection of the the draw framebuffer bounds and the dest region, and its corresponding sample point is in the intersection of the read framebuffer and the source region ([x0, x1] x [y0, y1]). sampling at x0 + y0 will access source pixels at position (x0 - 0.5, y0 - 0.5) or clamp to edge.

My take on the clarification:
A destination pixel is written only if its corresponding sample point is included both in the source region and the source framebuffer bounds.